.github/fabricbot.json

```diff
@@ -2196,7 +2196,599 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "isAction",
+              "parameters": {
+                "user": "roubachof",
+                "action": "opened"
+              }
+            },
+            {
+              "operator": "or",
+              "operands": [
+                {
+                  "name": "isActivitySender",
+                  "parameters": {
+                    "user": "roubachof"
+                  }
+                },
+                {
+                  "name": "isActivitySender",
+                  "parameters": {
+                    "user": "davidbritch"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        "eventType": "issue",
+        "eventNames": [
+          "issues",
+          "project_card"
+        ],
+        "taskName": "Add 'i/great-reporter' when issue is opened by an author we know opens high quality issues",
+        "actions": [
+          {
+            "name": "addLabel",
+            "parameters": {
+              "label": "i/great-reporter"
```

Comment:
Added the `i/great-reporter` label for issue reporters who have a good track record authoring issues on our repo.